lib/openshift/node.rb

```diff
@@ -255,5 +255,14 @@ def max_pod_count_schedulable(user: nil, cached: true, quiet: false, **pod_reque
       return remaining_pods_count if pod_requests.empty?
       return [ max_pod_count_capacity(user: user, cached: false, quiet: quiet, **pod_requests), remaining_pods_count].min
     end
+
+    def images(user: nil, cached: true, quiet: false)
```

Comment:
you can rename this to `private def raw_images(...)` and add a method `def images(...)` like in 
```
    def images(user: nil, cached: true, quiet: false)
      unless cached && props[:images]
        props[:images] = raw_images(user: user, cached: cached, quiet: quiet).map do |rimage|
          Image.new(name: rimage['names'].first, env: @owner.env)
        end
      end
      return props[:images]
    end
```
My concern though is whether there images are OpenShift resources of just some list of images that do not exist in the environment in any form. Do you know this? I assume they should exist.